### PR TITLE
Fixed a few additional typos

### PR DESCRIPTION
**Pull request by rmartinezduque**
Sorry, I just found a few more typos. They all should be fixed now! :D

### CODE REVIEW

**Review by juanmaguitar — APPROVED** [review]

**Comment by juanmaguitar** [comment]
Approved and Merged
Thanks @rmartinezduque

File: docs/contributing/README.md [hunk 1]
```diff
@@ -13,10 +13,10 @@ As you may know, Frontity is an open source project, available and free for ever
 Here are some more good reasons to contribute to open source:
 
 * **Build new connections**: an open source community usually consists of people sharing similar interests, so contributing to a project will allow you to connect with amazing people.
-* **Improve your skills and competences**: if you are looking for practice, there is always a task for you in open source projects. This will allow you improve your knowledge.
-* **Learn from mentors and teach others**: collaborating with others on a shared project means you'll have to ask people for help, as well as share your solutions and learnings with others. This can an be a fulfilling activity for everyone involved.
+* **Improve your skills and competences**: if you are looking for practice, there is always a task for you in open source projects. This will allow you to improve your knowledge.
```
Comment (mburridge) [inline docs/contributing/README.md:16]: ~~if you are looking for practice~~ → if you want to practice your [development/programming] skills

Comment (rmartinezduque) [reply docs/contributing/README.md:16]: @mburridge could you change this yourself? Or do you want me to submit a PR? I'm not sure how to proceed here.

Comment (mburridge) [reply docs/contributing/README.md:16]: @rmartinezduque I've changed it.

Comment (rmartinezduque) [reply docs/contributing/README.md:16]: Thanks @mburridge.

**Pull request merged**